lib/suse-cloud-upgrade-4-to-5-post

```diff
@@ -67,6 +67,27 @@ rcchef-solr start
 rcchef-expander start
 rcchef-server start
 
+# Before reinstalling the barclamps, set the horizon session timeout to a working default, since keystone barclamp will check it immediately on apply
+proposals=$(crowbar nova_dashboard proposal list)
+if [ "$proposals" != "No current proposals" ]; then
+  echo_summary "Setting default Horizon session timeout."
+  work_dir="${UPGRADE_DATA_DIR}/nova_dashboard"
+  mkdir -p $work_dir
+
+  for proposal in $proposals; do
+    # We only need to change and activate the default proposal
+    if [ "$proposal" != "default" ]; then
+      continue
+    fi
+
+    proposal_json="$work_dir/${proposal}.json"
+    crowbar pacemaker proposal show $proposal > "$proposal_json"
+    $json_edit $proposal_json -a "attributes.nova_dashboard.session_timeout" -v "240" 
+    crowbar nova_dashboard proposal edit $proposal --file $proposal_json || true
+
+    crowbar nova_dashboard proposal commit $proposal  
+  done
+fi
```

Comment:
The whole block is too early: the crowbar service isn't started, so the API calls won't work.